src/Options/OptionsDetails.hpp

```diff
@@ -169,12 +170,43 @@ struct print_impl {
 template <typename Tag, typename OptionList>
 struct print_impl<Tag, OptionList,
                   Requires<tmpl::list_contains_v<OptionList, Tag>>> {
-  template <typename LocalTag, Requires<has_default<LocalTag>::value> = nullptr>
+  template <typename LocalTag,
+            Requires<has_default<LocalTag>::value and
+                     not tt::is_a_v<std::unique_ptr, typename LocalTag::type>> =
+                nullptr>
   static std::string print_default() noexcept {
     std::ostringstream ss;
     ss << "default=" << std::boolalpha << LocalTag::default_value();
     return ss.str();
   }
+  template <
+      typename LocalTag,
+      Requires<has_default<LocalTag>::value and
+               tt::is_a_v<std::unique_ptr, typename LocalTag::type>> = nullptr>
+  static std::string print_default() noexcept {
+    std::ostringstream ss;
```

Comment:
Old legacy code that should all be revamped with some of those utilities. I opted for conformance to what is in the file rather than updating it all :man_shrugging:

Reply:
fair enough.